scripts/create-docs.sh

```diff
@@ -1,9 +1,30 @@
 #!/usr/bin/env bash
 
 TEMP_DIR=$(mktemp -d)
+if [[ $1 == "UEFI" ]]; then
+    IS_UEFI=true
+    echo "Generating UEFI version of the documentation"
+elif [[ $1 == "DCU" ]]; then
+    IS_UEFI=false
+    echo "Generating DCU version of the documentation"
+else
+    echo "Please provide version of the documentation: (UEFI/DCU)"
+    echo "Exit"
+    exit 0
+fi
 
 cat keywords.robot > "$TEMP_DIR/all-keywords.robot"
-find lib -name "*.robot" -exec cat {} + >> "$TEMP_DIR/all-keywords.robot"
+find lib -name "*.robot" -type f | while read -r file; do
```

Comment:
Maybe instead of seperating the keywords into two seperate documentations they could be shown together by prefixing the library name to the keyword name, just like the robot framework does when there are ambiguous keyword definitions. 

That would require recognizing the keyword names in their definitions in the contents of the file. With that appending the basename of the library file to every one of them in the library file would be easy. I think some code that could help with parsing the files should exist in the robot framework python package.

If not then maybe the keywords could be extracted and replaced using a regex like this? `^[A-Za-z0-9_\ ]+$` The keywords should always start from the first column, should and no other text should be after it. The keywords also only appear after the `*** Keywords ***` line. To make this method work only the contents after the `*** Keywords ***` line should be appended to the `all-keywords` file, assuming not including the `*** Settings ***` and other sections won't break libdoc.

I believe that if doing it in a more generic way, like adding the prefixes, would be more future-proof and won't require maintaining multiple documentations.

Reply:
@philipandag your request should be implemented in current version, can you check again and confirm if it's ok?